packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentationTemplate/CustomerSegmentationTemplate.ts

```diff
@@ -51,6 +51,11 @@ export interface CustomerSegmentationTemplateProps {
   templateQueryToInsert?: string;
   /* List of customer standard metafield used in the template's query. */
   standardMetafieldDependencies?: CustomerStandardMetafieldDependency[];
```

Comment:
@LTiger14 we are going to keep this field until we swap it out in segmentation templates and web.